dockerfiles/steps/step-bake.bash

```diff
@@ -19,7 +19,7 @@ for collection in "$IO_ASSEMBLED/"*.assembled.xhtml; do
     if [[ -f "$style_file" ]]
         then
             export VERBOSE=$TRACE_ON
-            $COOKBOOK_ROOT/bake -b "$style_name" -i "$IO_ASSEMBLED/$slug_name.assembled.xhtml" -o "$IO_BAKED/$slug_name.baked.xhtml"
+            $COOKBOOK_ROOT/bake -b "$style_name" -i "$IO_ASSEMBLED/$slug_name.assembled.xhtml" -o "$IO_BAKED/$slug_name.baked.xhtml" -r "_attic/IO_RESOURCES/"
```

Comment:
I think this will only work on local development... Can you try this instead?

```suggestion
            $COOKBOOK_ROOT/bake -b "$style_name" -i "$IO_ASSEMBLED/$slug_name.assembled.xhtml" -o "$IO_BAKED/$slug_name.baked.xhtml" -r "$IO_RESOURCES"
```